### ISSUE TITLE
Dizin Sayfası boş geliyor.

### ISSUE THREAD

**Issue by ramazansancar** [yazbel/python-istihza#201]
Kontrol edebilir misiniz?
https://python-istihza.yazbel.com/genindex.html

**Comment by scorpionabid** [comment]
Evet. Boş.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Ramazan SANCAR ***@***.***>
Sent: Thursday, September 29, 2022 4:41:31 PM
To: yazbel/python-istihza ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [yazbel/python-istihza] Dizin Sayfası boş geliyor. (Issue #201)


Kontrol edebilir misiniz?
https://python-istihza.yazbel.com/genindex.html

**Comment by ramazansancar** [comment]
Boş mu olması gerekiyor?

**Comment by EkremDincel** [comment]
Yanlış hatırlamıyorsam projeyi Sphinx'in 4.2.0 versiyonuna taşıyıp [template](https://github.com/yazbel/python-istihza/tree/master/source/_theme/templates)leri değiştirince olmuştu bu. [Sphinx konfigürasyonu](https://github.com/yazbel/python-istihza/blob/master/source/conf.py) ile uğraşıp düzeltmeye çalışmıştım ama bir yere varamayınca görmezden gelmiştim.

Projenin dizini zaten ana sayfada çıkıyor. Mevzubahis dizin sayfasının boş olmasının bir zararı yok ama gereksiz bir sayfa, ana sayfanın buraya link vermesine de gerek yok. Sphinx konfigürasyonu ile bu linki ana sayfadan silebiliyorsanız pull request'i kabul ederim.

**Comment by ramazansancar** [comment]
Tabi kısa sürede göndermiş olurum

**Comment by ismailarilik** [comment]
Uzun süre aktif olmadığı için kapatıldı.

**Comment by EkremDincel** [comment]
@ismailarilik Bahsedilen durum devam ediyor. Neden kapadınız anlamadım. Benim bildiğim issue'ler çözülmediği sürece kapatılmaz. Hem bizim problemleri takip etmemiz hem de projeye yardım etmek isteyenlerin görmesi için açık kalması daha sağlıklı olur düşüncesindeyim.

**Comment by ismailarilik** [comment]
Doğru. Öyle düşünmemiştim, pardon. Bazı projelerde uzun süre yorum yapılmayan issue'lar kapatılıyor. Ama başkalarının görmesi açısından dediğin mantıklı. Çok issue olursa önceliklendirme gerekir ama daha o noktada değiliz.
